### ISSUE TITLE
The new README template should have a blank line

### ISSUE THREAD

**Issue by stuartpb** [isaacs/github#338]
When generating a README for new repositories, while I like the new switch to an ATX-style `#` for the heading, it's wrong to *immediately* follow that heading line with the repo's description.

Headings are block-level content, and traditional lazy-continuation rules for blocks (depending on which Markdown renderer is interpreting the file) will roll a line immediately following the line before it into the previous line, counting it as the same block. (Git has the same behavior when writing a commit message: if you type on the second line, it will be treated as a continuation of the subject, rather than the start of the body.)

tl;dr, this:

```
# {{reponame}}
{{description}}
```

should be this:

```
# {{reponame}}

{{description}}
```

(I messed up the space between the `#` and the title in the version I sent to GitHub.)

**Comment by stuartpb** [comment]
Okay, so I just [tested this], and technically no interpreter continues the heading onto the second line.

I'm still right about the general block spacing convention used by GitHub, though.
